packages/shared/src/components/squads/SquadPageHeader.tsx

```diff
@@ -27,6 +34,18 @@ interface SquadPageHeaderProps {
 const MAX_WIDTH = 'laptopL:max-w-[38.5rem]';
 const Divider = classed('span', 'flex flex-1 h-px bg-border-subtlest-tertiary');
 
+interface SquadStatProps {
+  count: number;
+  label: string;
+}
+
+const SquadStat = ({ count, label }: SquadStatProps) => (
+  <span className="flex flex-row text-text-tertiary typo-footnote">
+    <sup className="mr-1 font-bold text-text-primary typo-subhead">{count}</sup>
```

Comment:
This count looks odd in your screenshots, is that intended? (like weirdly placed)

Reply:
I used the native `sup` HTML element, we can maybe hack it with `position` or something. Let me check.

Reply:
I don't really liked it either, checked in designs as well when reviewing this..

Reply:
Oh it's actually like that in design how odd!
Did we verify this lol?

Reply:
![Screenshot 2024-04-26 at 10 33 18 PM](https://github.com/dailydotdev/apps/assets/13744167/3d779680-97aa-4c23-9796-9b109758c103)